sound/soc/intel/boards/sof_sdw_rt711.c

```diff
@@ -151,6 +165,7 @@ int sof_sdw_rt711_init(const struct snd_soc_acpi_link_adr *link,
 	if (ret < 0)
 		return ret;
 
+	info->is_probe = true;
```

Comment:
@bardliao At least in this case, this "is_probe" state variable is not realy needed. rt711_exit() can handle the case where device is not probed (sdw_dev will be null and no cleanup done).

Reply:
My point is that the codec could be probed, but not used by the machine driver. In such case, we can still find sdw_dev, but we don't want to call device_remove_properties since sof_sdw_rt711_init() is not called, too.